source/_patterns/04-pages/inside-elife-article.mustache

```diff
@@ -20,9 +20,9 @@
 
       <div class="wrapper wrapper--content">
 
-        <div class="grid">
+        <div class="content-container-grid content-header__one-column">
```

Comment:
No need for the content-header__one_column class here!

We need to borrow more from the digest template. I think a short pairing session could get this into the shape it needs to be. Let's try tomorrow morning or Friday!

Reply:
We should refer to:
- https://github.com/elifesciences/pattern-library/pull/1140/files
- https://github.com/elifesciences/journal/pull/1629/files

It may be that we need to adjust the logic for digests so that we can serve digests and inside-elife with a common logic.